server/src/visitors/YmlClassVisitor.ts

```diff
@@ -0,0 +1,42 @@
+import { CompletionItem, CompletionItemKind } from "vscode-languageserver";
+import {
+  ClassAttributeDeclarationContext,
+  ClassDeclarationIntroContext,
+  MethodDeclarationContext,
+} from "../YmlToBdlParser";
+import { createNewCompletionItem } from "./utils";
+import YmlToBdlBaseVisitor from "./YmlToBdlBaseVisitor";
+
+export class YmlClassVisitor extends YmlToBdlBaseVisitor {
+  private classId: string;
+
+  constructor(completionItems: CompletionItem[]) {
+    super(completionItems);
+  }
+
+  public visitMethodDeclaration(node: MethodDeclarationContext): any {
+    createNewCompletionItem(
+      this.completionItems,
+      node.methodIntro().ymlId(),
+      node.field(),
+      CompletionItemKind.Method,
```

Comment:
Is that where the numbers seen in the tests come from? Is there a way to get something roughly equivalent to an enum value's name in Java? ⇒ `Method` rather than an integer?